client/my-sites/plugins/plugins-list/plugins-list-dataviews.tsx

```diff
@@ -0,0 +1,286 @@
+			},
+		],
+		[]
+	);
+
+	const actions = [
+		{
+			id: 'manage-plugin',
+			href: `some-url`,
+			callback: ( plugins: Array< Plugin > ) => {
+				plugins.length && navigate( '/plugins/' + plugins[ 0 ].slug );
+			},
+			label: translate( 'Manage Plugin' ),
+			isExternalLink: true,
+			isEnabled: true,
+			supportsBulk: false,
+		},
+		{
+			id: 'activate-plugin',
+			href: `some-url`,
+			callback: ( plugins: Array< Plugin > ) => {
+				bulkActionDialog( PluginActions.ACTIVATE, plugins );
+			},
+			label: translate( 'Activate' ),
+			isExternalLink: true,
+			isEnabled: true,
+			supportsBulk: true,
+			icon: <Icon icon={ link } />,
+		},
+		{
+			id: 'deactivate-plugin',
+			href: `some-url`,
+			callback: ( plugins: Array< Plugin > ) => {
+				bulkActionDialog( PluginActions.DEACTIVATE, plugins );
+			},
+			label: translate( 'Deactivate' ),
+			isExternalLink: true,
+			isEnabled: true,
+			supportsBulk: true,
+			icon: <Icon icon={ linkOff } />,
+		},
+		{
+			id: 'enable-autoupdate',
+			href: `some-url`,
+			callback: ( plugins: Array< Plugin > ) => {
+				bulkActionDialog( PluginActions.ENABLE_AUTOUPDATES, plugins );
+			},
+			label: translate( 'Enable Autoupdate' ),
+			isExternalLink: true,
+			isEnabled: true,
+			supportsBulk: true,
+		},
+		{
+			id: 'disable-autoupdate',
+			href: `some-url`,
+			callback: ( plugins: Array< Plugin > ) => {
+				bulkActionDialog( PluginActions.DISABLE_AUTOUPDATES, plugins );
+			},
+			label: translate( 'Disable Autoupdate' ),
+			isExternalLink: true,
+			isEnabled: true,
+			supportsBulk: true,
+		},
+		{
+			id: 'remove-plugin',
+			href: `some-url`,
+			callback: ( plugins: Array< Plugin > ) => {
+				bulkActionDialog( PluginActions.REMOVE, plugins );
+			},
+			label: translate( 'Remove' ),
+			isExternalLink: true,
+			isEnabled: true,
+			supportsBulk: true,
+			icon: <Icon icon={ trash } />,
+		},
+	];
+
+	// Set initial state for the data views
+	const pluginsPerPage = 15;
+	initialDataViewsState.perPage = pluginsPerPage;
+	initialDataViewsState.search = initialSearch;
+	initialDataViewsState.fields = [ 'plugins', 'sites', 'update' ];
+	const [ dataViewsState, setDataViewsState ] = useState( initialDataViewsState );
+
+	// When search changes, notify the parent component
+	useEffect( () => {
+		onSearch && onSearch( dataViewsState.search || '' );
+	}, [ dataViewsState.search, onSearch ] );
+
+	const { data, paginationInfo } = useMemo( () => {
+		const result = filterSortAndPaginate( currentPlugins, dataViewsState, fields );
+
+		// Add all statuses to the plugin object so we can display them in the DataViews
+		result.data.forEach( ( plugin ) => {
+			plugin.allStatuses = allStatuses.filter( ( status ) => status.pluginId === plugin.id );
+		} );
+
+		return {
+			data: result.data,
+			paginationInfo: result.paginationInfo,
+		};
+	}, [ currentPlugins, dataViewsState, fields, allStatuses ] );
+
+	return (
+		<div className="referrals-details-table__container redesigned-a8c-table">
```

Comment:
Ideally, we should have no style overrides. Maybe just colors 😉 

This might be worth a more central audit across the rest of a4a.

Reply:
Good catch! This one was part of the initial tests, and we [don't need them anymore](https://github.com/Automattic/wp-calypso/pull/95561/commits/c289c918e08d09c4c211583a6c7037feb34fdc20).